packages/dds/sequence/src/sequence.ts

```diff
@@ -68,7 +68,7 @@ const contentPath = "content";
 /**
  * Events emitted in response to changes to the sequence data.
  *
- * @remarks
+ * @remarks @public
```

Comment:
This syntax is wrong. We'll need to move the `@public` tag to the bottom of the comment.